readthedocs-theme/templates/article.html

```diff
@@ -0,0 +1,49 @@
+{% extends "base.html" %}
+{% block html_lang %}{{ article.lang }}{% endblock %}
+
+
+{% block title %}{{ article.title|striptags  }} - {{ SITENAME }}{% endblock %}
+
+
+{% block content %}
+
+
+  <!-- Article -->
+  <section class="ui vertical segment padded" id="article">
+    
+    <div class="ui container">
+      <div class="ui grid centered">
+        <div class="column twelve wide computer fourteen wide tablet sixteen wide mobile">
+
+
+          <h1 class="ui header">
+            <span class="ui header big">{{ article.title }}</span>
+          </h1>
+
+          <div class="ui basic vertical segment clearing">
+            <div class="ui fitted divider"></div>
+            <div class="ui basic vertical segment left floated">{% for author in article.authors %}{{ author }}{% if not loop.last %}, {% endif %}{% endfor %}</div>
+            <div class="ui basic vertical segment right floated"><time datetime="{{ article.date.isoformat() }}">{{ article.locale_date }}</time></div>
+          </div>
+
+          {% if article.image %}
+              <img class="ui fluid image" src="{{ article.image }}">
+              {% if article.image_credit %}
+                <div class="ui basic vertical segment center aligned">
+                  <p class="ui"><i>{{ article.image_credit }}</i></p>
+                </div>
+              {% endif %}
+          {% endif %}
+
+          <div class="ui basic vertical segment padded">
+            {{ article.content }}
```

Comment:
So, I noticed a display issue that I was expecting on the blog posts. The Pelican HTML output is not going to have the structure that would be expected -- so for example headings nested in `div.section` do not have a top margin.

![image](https://user-images.githubusercontent.com/1140183/152935144-154b77a0-777c-415c-80c6-284880222c1e.png)

At very least, we should catch this and apply additional top padding. It might also help to be testing against more that what we have for test blog posts. Currently, we just have two short blog posts we're testing on.

Reply:
I haven't started messing with custom styles here but both the blog post and the blog index pages could definitely use some custom CSS. I opened an issue for the blog post here #67; had already one open for the blog index #61

Reply:
* PR #70 fixes this and closes #67

Reply:
There might be more work to do later, but this PR should probably address some of the basic display issues on blog content. Not having more test blog posts here is probably delaying finding these issues